vitrina/datasets/views.py

```diff
@@ -1047,6 +1050,8 @@ def get_form_class(self):
 
         if subclass.name == DCATResourceSubclass.SERVICE:
             return ServiceResourceForm
+        if subclass.name == DCATResourceSubclass.INFORMATION_SYSTEM:
```

Comment:
Optional (for readability)
```suggestion
        elif subclass.name == DCATResourceSubclass.INFORMATION_SYSTEM:
```

Reply:
<img width="931" height="689" alt="Screenshot 2025-08-13 at 12 17 06" src="https://github.com/user-attachments/assets/fed109b8-c8a1-4b2e-bb13-ef9e58936818" />

This feels like walking in circles 🔁 😄